packages/components/backtop/__tests__/backtop.spec.tsx

```diff
@@ -1,28 +1,30 @@
-import { nextTick } from 'vue'
+import { nextTick, type VNode } from 'vue'
```

Comment:
Please import type individually:

```ts
import type { VNode } from 'vue'
```

put this line after line#5

Reply:
ok

Reply:
Maybe you reverted this changes?